src/types.ts

```diff
@@ -40,6 +40,19 @@ export type PackedClaims = {
 
 export type Hasher = (data: string) => Promise<string>;
 export type SaltGenerator = (size) => string;
+
+export interface IssueSDJWTOptions {
+  header?: JWTHeaderParameters;
+  payload: JWTPayload;
+  disclosureFrame: DisclosureFrame;
+  alg: string;
+  getHasher: (hash_alg: string) => Promise<Hasher>;
+  hash_alg?: string;
```

Comment:
@ragnika this should be typed, something similar to this: https://github.com/berendsliedrecht/sd-jwt-ts/blob/main/src/hasherAlgorithm.ts
And maybe function inside should have additional check so it's from this list. Now user can pass whatever there and that will end up in an sd-jwt that is impossible to process later.